platform.md

```diff
@@ -122,6 +131,11 @@ The buildpacks directory MUST contain unarchived buildpacks such that:
 - Each top-level directory is a buildpack ID.
 - Each second-level directory is a buildpack version.
 
+## Exported Image
+The Exported image containing the compiled application is generated by the Build Image during the [Export Phase](#export-phase).
+
+If the `/cnb/lifecycle/exporter` is provided with a `project-metadata.toml` file in the Platform directory, the lifecycle MUST add a `io.buildpacks.project` image config label to the Exported image. This label value is encoded JSON format of the `project-metadata.toml` contents, as described in the [data-format](#data-format) section.
```

Comment:
@ekcasey any thoughts about `io.buildpacks.project` to match the other labels? I noticed this key was already added to the lifecycle.

Reply:
@sclevine that suggestion makes sense to me

Reply:
here ya go - buildpacks/lifecycle#257